### PR TITLE
Add a section for manual importing 2020 projects

### PR DESCRIPTION
**Pull request by sciencewhiz**
Includes the steps to update the files manually with appropriate
warnings

### CODE REVIEW

File: source/docs/software/vscode-overview/importing-gradle-project.rst [hunk 1]
```diff
@@ -1,12 +1,17 @@
 Importing a Gradle Project
 ==========================
 
-Due to changes in the project, it is necessary to update the build files for a previous years Gradle project. To make it easy for teams to import previous years gradle projects into the current year's framework, WPILib includes a wizard for importing previous years projects into VS Code. This will generate the necessary gradle components and load the project into VS Code. In place upgrades are not supported. The importer automatically imports the old command framework 3rd party library.
+Due to changes in the project, it is necessary to update the build files for a previous years Gradle project.
+
+Automatic import
```
Comment (Daltz333) [inline source/docs/software/vscode-overview/importing-gradle-project.rst:6]:
```
Automatic Import
```

File: source/docs/software/vscode-overview/importing-gradle-project.rst [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+----------------
+
+To make it easy for teams to import previous years gradle projects into the current year's framework, WPILib includes a wizard for importing previous years projects into VS Code. This will generate the necessary gradle components and load the project into VS Code. In place upgrades are not supported. The importer automatically imports the old command framework 3rd party library.
```
Comment (Daltz333) [inline source/docs/software/vscode-overview/importing-gradle-project.rst:9]: Unrelated to this PR, but do we still want this to be the intended behavior?

File: source/docs/software/vscode-overview/importing-gradle-project.rst [hunk 3]
```diff
@@ -34,14 +39,247 @@ The gradle project will be upgraded and copied into the new project directory. Y
...
 
+Manual Import Process (2020 to 2021)
+------------------------------------
+
+As there were minimal changes to the gradle templates in 2021, it is possible to manually import a project from a 2020 version of GradleRIO to 2021, which may be convenient if you've heavily customized your build.gradle file. This method does *not* work for importing a 2019 project, and will not work for future years. There are two files that need to be updated.
+
+wpilib_preferences.json
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Open ``.wpilib\wpilib_preferences.json`` and change the projectYear to 2021
```
Comment (Daltz333) [inline source/docs/software/vscode-overview/importing-gradle-project.rst:61]:
```suggestion
Open ``.wpilib\wpilib_preferences.json`` and change the ``projectYear`` to ``2021``
```

**Review by Daltz333 — CHANGES_REQUESTED** [review]

File: source/docs/software/vscode-overview/importing-gradle-project.rst (hunk 2 above)
Comment (sciencewhiz) [reply source/docs/software/vscode-overview/importing-gradle-project.rst:9]: We actually changed just changed this, will update the PR

**Review by AustinShalit — APPROVED** [review]

**Pull request merged**